### ISSUE TITLE
Make WindNet India batches using MO global data 

### ISSUE THREAD

**Issue by Sukh-P** [openclimatefix/PVNet#260]
Make batches/samples for WindNet model which uses the MO global NWP as an additional input

**Comment by Sukh-P** [comment]
One set of batches have already been created which include MO data, a second set is also being created which fills in some gaps in GFS to include data as far back as June 2019.

**Comment by Sukh-P** [comment]
The second set of batches have been paused, the validation set has been created but the training set creation has been paused due to a long running time/resource constraints. Would be good to create these batches in the future to experiment with more samples from more years for WindNet.
